apps/system/js/operator_variant_handler.js

```diff
@@ -300,7 +303,8 @@
           'ril.dun.passwd': 'password',
           'ril.dun.httpProxyHost': 'proxy',
           'ril.dun.httpProxyPort': 'port',
-          'ril.dun.authtype': 'authtype'
+          'ril.dun.authtype': 'authtype',
+          'ril.dun.mtu': 'mtu'
```

Comment:
add test in https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L75